src/Components/allWebsite.jsx

```diff
@@ -0,0 +1,27 @@
+import React,{useState,useEffect} from 'react';
+import {GetWebsitesApi} from '../Api/api'
+
+
+export const WebsiteList = () => {
+  const [websites, setWebsites] = useState([]);
+
+	useEffect(() => {
+		const fetchData = async () => {
+			const data = await GetWebsitesApi();
```

Comment:
put attention that GetWebsitesApi will return null (and I really dont like it) when there is an error (the catch block) in such a case, this code and the code below will throw an error (websites.map on null)